synapseutils/sync.py

```diff
@@ -61,6 +62,8 @@ def syncFromSynapse(syn, entity, path=None, ifcollision='overwrite.local', allFi
     :param followLink:  Determines whether the link returns the target Entity.
                         Defaults to False
 
+    :param manifest:    Determines whether creating manifest file automatically.
```

Comment:
Can you document the optional values here ("all", "root", "suppress") as part of the doc string?

Reply:
Also it would be good to validate the value passed in is one of the options. The command line call will do that automatically when this is called from the command line, but if invoked programmatically there is no check.